src/api/app/views/layouts/webui/_status_message_announcement.html.haml

```diff
@@ -0,0 +1,12 @@
+- if current_announcement
+  .row.justify-content-center#status-message-announcement
+    .col-12
+      .alert.fade.show.alert-notice
+        .row
+          .col-md-10
+            %i.fa.fa-bullhorn.text-info
+            = truncate(current_announcement.message, length: 80)
```

Comment:
Why truncate and how can I see the rest of the message? What about markdown?

Reply:
We need at least the ability to include links.

Reply:
If we don't truncate, then we can render as markdown again. Yes!.